src/reducers/waypoint-reducer.ts

```diff
@@ -54,19 +54,27 @@ export const {
 /**
  * Checks wheter route ID has been initialized. If so, gets location and MNC code and creates
  * a new waypoint object which will stored into localdevices `WaypointState`
- * Sends also waypoints to the server, if there are more than 5 pending waypoints
+ * Sends waypoints to server according to the sending interval or immediately if
+ * was disconnected and is connected.
  * @returns dispatch method to update `WaypointState`
  */
+let wasOffline = false;
+let sendTicker = 0;
 export const storeAndSendWaypoints = () => {
 	return async (dispatch: AppDispatch, getState: () => ReduxState) => {
 		const routeId = getState().waypoints.routeId;
```

Comment:
JS-Doc refers now to the `wasOffline` variable instead of `storeAndSendWaypoints`. Perhaps move the variables above JS-Doc?